packages/@react-spectrum/calendar/src/CalendarBase.tsx

```diff
@@ -60,24 +61,49 @@ export function CalendarBase<T extends CalendarState | RangeCalendarState>(props
     timeZone: state.timeZone
   });
 
-  let titles = [];
   let calendars = [];
   for (let i = 0; i < visibleMonths; i++) {
-    titles.push(
-      <div key={i} className={classNames(styles, 'spectrum-Calendar-monthHeader')}>
+    let d = currentMonth.add({months: i});
+    calendars.push(
+      <div key={i} className={classNames(styles, 'spectrum-Calendar-month')}>
+        {/* Put the heading first so it is the first thing touch screen reader users encounter. */}
+        <h2
```

Comment:
Would it make sense to add the Calendar's `aria-label` to this heading so that a touch screen reader user gets the Calendar label announced sooner rather than announced later when they swipe through to the first date of the Calendar? If so, maybe an argument could be made to add other statuses (such as the Calendar's validation state) to this header.

Not too chuffed either way since it isn't a ton of swiping to enter the calendar grid itself.

Reply:
Well, we couldn't add it to this one because this is visible. I suppose we could always use a visually hidden heading rather than only when there are multiple months. Not sure...

Reply:
Actually I think this is a good idea because VoiceOver on iOS doesn't seem to announce the aria-label of the grid at all...